api/envoy/config/route/v3/route_components.proto

```diff
@@ -320,9 +320,29 @@ message WeightedCluster {
 
     reserved "per_filter_config";
 
+    // Only one of *name* and *cluster_header* may be specified.
+    // [#next-major-version: Need to add back the validation rule: (validate.rules).string = {min_len: 1}
```

Comment:
I think you need a `]` at the end of this line.

Reply:
oops, missed `]` on this code comment. Thanks for catching it!
done.